Show Kenya subnational data

KhadijaMahanga opened the pull request:
## Description
- [x] Add Sources for new takwimu charts (`kenya subnational data`) defined on wordpress engine
- [x] `drawChildren = true`  mapit Kenya

Fixes [#168566062](https://www.pivotaltracker.com/story/show/168566062)

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Screenshots
![Screenshot from 2019-10-30 17-49-03](https://user-images.githubusercontent.com/7962097/67868979-c9479800-fb3d-11e9-8fe4-9a57e176faa9.png)


## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

kilemensi commented on src/data/sources.json:
```diff
@@ -283,6 +283,80 @@
+            },
+            "allHivPrevalences": {
+                "source": {
+                    "href": "https://nacc.or.ke/wp-content/uploads/2018/12/HIV-estimates-report-Kenya-20182.pdf",
```
Linking directly to PDF _ain't_ cool.

KhadijaMahanga replied:
@kilemensi  I will ping soila to upload this to openAfrica

KhadijaMahanga replied:
updated this

kilemensi commented on src/pages/Profile.js:
```diff
@@ -122,6 +122,7 @@ function Profile({ chartDefinitions }) {
     table => {
       const source =
         chartSources[country.slug][profiles.profile.geoLevel][table];
+      console.log(table);
```
Clean up?

KhadijaMahanga commented:
>  @KhadijaMahanga
> 
> Few questions/observation though (need not be addressed in this PR):
> 
> 1. Sections: They just appear to be alphabetically ordered but this is not how they should be e.g. Demographics should be one of the first... Can we reorder?
> 2. The NumberVisuals description appears to be too big and not so elegant
>    ![Screenshot from 2019-10-31 07-43-07](https://user-images.githubusercontent.com/1779590/67919779-ccc93680-fbb2-11e9-90f8-b38f6d239ada.png)

@kilemensi 
1. I have noticed the ordering of sections, I have created a story on the Dashboard to order the sections.

2. I have renamed the specific long Number visual chart title. Plus charts definition on dashboard were missing `subtitle` which is the title for number visual charts, so I have updated that on the dashboard.
I will write another bug story on PT for HURUmap-UI to fix `word-break` cause at the moment is it breaking between characters.

KhadijaMahanga commented:
I added a task to this famous story https://www.pivotaltracker.com/story/show/169120973 :smile:

karimkawambwa commented:
@KhadijaMahanga 

> I will write another bug story on PT for HURUmap-UI to fix word-break cause at the moment is it breaking between characters, and I can't send custom style

The character break is intentional as it causes flex to not work appropriately.

KhadijaMahanga commented:
> @KhadijaMahanga
> 
> > I will write another bug story on PT for HURUmap-UI to fix word-break cause at the moment is it breaking between characters, and I can't send custom style
> 
> The character break is intentional as it causes flex to not work appropriately.

@karimkawambwa I see.. but I think we still need to look into it again, cause words are breaking..and doesn't look so elegant (in Kilemensi's voice)

karimkawambwa commented:
😆 
> (in Kilemensi's voice)

![image_from_ios-1](https://user-images.githubusercontent.com/4308339/68026383-a1cd0880-fcc0-11e9-9152-8fccd28e4c5b.jpg)


@KhadijaMahanga let's get a different eye on that. This was my solution to the long words causing inconsistent `InsightConatiner` flex. Perhaps Monday you can start looking into it. Its the Hurumap InsightContainer, just turn remove `word-break` and use really long words that can't be broken up.
